test/controllers/calls_controller_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'test_helper'
+
+class CallsControllerTest < ActionController::TestCase
+  include Devise::TestHelpers
+
+  def setup
+    @user = build :user
+    sign_in @user
+    @case = create :case
+    @call = build :call
+  end
+
+  describe 'create method' do
+    before do
+      @call = attributes_for :call
+    end
+
+    it 'should create and save a new call' do
+      assert_difference 'Case.find(@case).calls.count', 1 do
+        post :create, status: @call.status, id: @case
+      end
+      assert_response :redirect
+    end
+
+    # If we have to look up the call through a case this test seems redundant
+    # I.e. I think the previous test also confirms the association
+    # it 'should associate the new call with a case' do
```

Comment:
Can you think of a better way to write the assert_includes statement? As is I think it would always pass if @case has any calls at all